### PR TITLE
koa changes

### PR DESCRIPTION
**Pull request by aaschaer**
Changes to allow the daemon to run outside of koa

### CODE REVIEW

File: globus_cw_daemon/daemon.py [hunk 1]
```diff
@@ -1,26 +1,17 @@
...
 
-import pykoa
-from pykoa.koaexception import script_exception_handler
-from pykoa.lib import daemon
-from pykoa.lib import cwlogs
-
-
-# Don't log unhandled exceptions to ourself - it hangs.
-pykoa.log_unhandled_to_cloudwatch = False
+import cwlogs
```
Comment (karlito-go) [inline globus_cw_daemon/daemon.py]: can this be globus_cw_daemon.cwlogs?

File: globus_cw_daemon/daemon.py [hunk 2]
```diff
@@ -29,18 +20,30 @@
...
-_do_local_log = pykoa.get_misc_conf("cwlogs.do_local_log", 0)
+_log = logging.getLogger(__name__)
 
 # Data shared with flush thread
 _g_lock = threading.Lock()
 _g_queue = []   # List of Events
 _g_nr_dropped = 0
 
 
+def _script_exception_handler(fn):
```
Comment (karlito-go) [inline globus_cw_daemon/daemon.py]: I don't think we need this; an unhandled exception should be logged by systemd, assuming we're not daemonizing.

File: globus_cw_client/client.py [hunk 3]
```diff
@@ -1,21 +1,30 @@
 """
 Python client API for cwlogs daemon
 """
-import time, socket, simplejson
+import time, socket, json
```
Comment (karlito-go) [inline globus_cw_client/client.py]: please test the performance of simplejson vs json.
if there's a big difference (due to one using a c module, which used to be the case), consider a:

try:
  import simplejson as json
except Importerror
  import json as json

Comment (aaschaer) [reply globus_cw_client/client.py]: They were about the same for python 3, but json was significantly faster for 2.7, which surprised me.

Comment (karlito-go) [reply globus_cw_client/client.py]: nuts.  I get the same results.  ok, stick with json.

**Pull request merged**